### PR TITLE
feat: projects page

### PR DESCRIPTION
**Pull request by armankir**
I added the projects page to the website.

This version is not responsive, it will require further refactoring.

image
image
image

### CODE REVIEW

**Comment by ErickSharp** [comment]
I have a branch set up to fix the scaling issues with the extended features section. Can't request changes on here so I might just open another pull request after this is merged. Message me on Discord if you want to get the code to put it in this single PR!

**Comment by ErickSharp** [comment]
I have a PR open to change the download section to be for the installer like you asked, we are going to have to figure something out as to how we will include the installer and the direct download links. I have something in mind like making a very thin bar for the direct downloads but above that is the Installer component.

**Comment by ErickSharp** [comment]
I like the idea and look of the plane, just with some minor tweaking. One thing that I have noticed is that the plane doesn't load in at once when you load up the page, it loads in a bit after.

**Comment by armankir** [comment]
> I like the idea and look of the plane, just with some minor tweaking. One thing that I have noticed is that the plane doesn't load in at once when you load up the page, it loads in a bit after.

Can't do anything about that. What kind of tweaking do you want? Request the changes.

File: src/components/projects/A32NX.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                        <a href="#download">
+                            <Button
+                                className="w-40 text-xl font-bold border-2 border-blue-light text-blue-light transition hover:bg-white hover:text-blue-light">Download
+                            </Button>
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <img draggable="false" className="absolute -right-1/3 transform scale-150 opacity-30 select-none" src={a32nx_wireframe} alt="" />
```
Comment (ErickSharp) [inline src/components/projects/A32NX.tsx:40]:
```suggestion
        <img draggable="false" className="absolute hidden md:block right-96 md:-right-1/3 transform scale-150 opacity-40 select-none" src={a32nx_wireframe} alt="" />
```
Make the wireframe hidden on mobile

**Review by ErickSharp — CHANGES_REQUESTED** [review]

File: src/components/projects/A32NX.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                        <a href="#download">
+                            <Button
+                                className="w-40 text-xl font-bold border-2 border-blue-light text-blue-light transition hover:bg-white hover:text-blue-light">Download
+                            </Button>
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <img draggable="false" className="absolute hidden md:visible -right-1/3 transform scale-150 opacity-30 select-none" src={a32nx_wireframe} alt="" />
```
Comment (ErickSharp) [inline src/components/projects/A32NX.tsx:40]:
```suggestion
        <img draggable="false" className="absolute hidden md:block -right-1/3 transform scale-150 opacity-30 select-none" src={a32nx_wireframe} alt="" />
```

**Review by ErickSharp — CHANGES_REQUESTED** [review]
oops, should be block and not visible; my bad.

**Comment by ErickSharp** [comment]
There’s an issue with the scaling for the extended features I didn’t notice, it’s pretty minor and I have a fix for it. It changes it to 3 per row at medium but then you will have 2 at the very bottom without a third one to complete the row so I don’t know if you would like the look of that.

**Review by NathanInnes — APPROVED** [review]

**Pull request merged**